bot.lua

```diff
@@ -4768,17 +4779,20 @@ commands["lua"] = {
 				assert(memberId, "Member ID can't be nil in discord.getMemberName")
 				memberId = tostring(memberId)
 
-				local member = message.guild:getMember(memberId)
+				local guild = message.guild or client:getGuild(channels["guild"])
+				local member = guild:getMember(memberId)
 				return member and member.name
 			end
 
 			ENV.discord.isMember = function(userId)
 				assert(userId, "Member id cannot be nil in discord.isMember")
-				return message.guild:getMember(userId) ~= nil
+				local guild = message.guild or client:getGuild(channels["guild"])
+				return guild:getMember(userId) ~= nil
 			end
 
 			ENV.discord.sendPrivateMessage = function(content)
 				assert(content, "Content cannot be nil in discord.sendPrivateMessage")
+				assert(message.guild, "Use discord.reply instead of discord.sendPrivateMessage when replying to a DM")
```

Comment:
server commands may reply in private, so just remove this line

Reply:
It will only trigger if it is trying to reply to a DM using this function. IMO it'd be better to just have discord.reply as it would delete the bot's messages if the author decides to delete the command.

Reply:
Nope. I want to allow DMs like that too :P Please, remove the line.